src/ports/nfts/queries.ts

```diff
@@ -455,9 +461,9 @@ function getNFTWhereStatement(nftFilters: GetNFTsFilters): SQLStatement {
 }
 
 function getRecentlyListedNFTsCTE(nftFilters: GetNFTsFilters): SQLStatement {
-  const FILTER_BY_OWNER = nftFilters.owner
-    ? SQL` owner_id IN (SELECT id FROM squid_marketplace.account WHERE address = ${nftFilters.owner.toLocaleLowerCase()}) `
-    : null
+  const ownerEthereumAddress = nftFilters.owner ? `${nftFilters.owner.toLocaleLowerCase()}-ETHEREUM` : null
```

Comment:
this change speeds the query up (it will be change once the new squid is released)